docs/detections/prebuilt-rules-management.asciidoc

```diff
@@ -108,18 +107,85 @@ NOTE: The *Rule Updates* tab doesn't appear if all your installed prebuilt rules
+
+** Address update conflicts: Find and address conflicts that <<resolve-reduce-rule-conflicts, need additional attention>>. 
++
+IMPORTANT: Elastic updates to a rule's type cannot be changed. Before updating the rule, duplicate it if you need to record changes that you made to other rule fields. 
+
+** Edit the final update: Change the update that will be applied to the field when you update the rule. To change the update, go to the *Final update* section, make your changes, and then save them.
 +
 [role="screenshot"]
-image::images/prebuilt-rules-update-diff.png[Prebuilt rule comparison,75%]
+image::images/prebuilt-rules-update-diff.png[Prebuilt rule comparison,85%]
++
 
-. Do one of the following to update prebuilt rules on the *Rules* page:
+. From the *Rule Updates* tab, do one of the following to update prebuilt rules:
 * Update all available rules: Click *Update all*.
-* Update a single rule: Click *Update rule* for that rule.
+* Update a single rule: Click *Update rule* for that rule. 
 * Update multiple rules: Select the rules and click *Update _x_ selected rule(s)*.
 +
 TIP: Use the search bar and *Tags* filter to find the rules you want to update. For example, filter by `OS: Windows` if your environment only includes Windows endpoints. For more on tag categories, refer to <<prebuilt-rule-tags>>.
+
+[float]
+[[rule-update-statuses]]
+==== Rule update statuses
+
+This table describes statuses that might appear for rule fields being updated.  
+
+[cols="2"]
+|===
+
+| *Ready for update*
+a| Displays when there are no conflicts to resolve. 
+
+Further action is not required for the field. It is ready to be updated.
+
+| *No update*
+a| Displays when the field is not being updated by Elastic, but the current field value differs from the original one. This typically happens when the field's value was changed after the prebuilt rule was initially installed.
+
+Further action is not required for the field. It is ready to be updated.
+
+TIP: You can still change the final field update, if needed. To do so, make your changes in the *Final update* section and save them.
+
+| *Review required*
+a| Displays when Elastic auto-resolves a conflict between the current field value and the value from the incoming Elastic update. 
+
+You must accept or edit the field's final update and save the changes. Refer to <<resolve-reduce-rule-conflicts>> to learn more about auto-resolved conflicts and how to reduce future conflicts.
+
+| *Action required*
+a| Displays when Elastic could not auto-resolve the conflict between the current field value and the value from the incoming Elastic update. 
+
+You must manually set and save the field's final update. Refer to <<resolve-reduce-rule-conflicts>> to learn more about conflicts that need manual fixes and how to reduce future conflicts.
+
+|===
+
+
+[float]
+[[resolve-reduce-rule-conflicts]]
+==== Resolve and reduce update conflicts
```

Comment:
This PR just got merged so I don't think it's officially made it back to `8.18` (yet) but we should probably add a little more documentation in this section covering some of the changes in https://github.com/elastic/kibana/pull/213285. Most notably that you **can** upgrade rules in bulk with auto-resolved conflicts and the additional column in the updates table we add that shows if a rule's conflict level. Let me know if you want to sync on that @nastasha-solomon

Reply:
Yeah, now that 213285 has been merged, I'll need to update these docs and maybe a few other areas. A quick sync would definitely be helpful for getting up to speed on recent changes. Thanks, @davismcphee.

Reply:
Partially addressed in [76fe435](https://github.com/elastic/security-docs/pull/6568/commits/76fe4359711efb160765be02325c265ccfca23e8).